turbo.json

```diff
@@ -72,6 +65,7 @@
     "POSTGRES_URL",
     "DATABASE_URL",
     "POSTGRES_PRISMA_URL",
-    "POSTGRES_URL_NON_POOLING"
+    "POSTGRES_URL_NON_POOLING",
+    "NEXT_PRIVATE_STRIPE_WEBHOOK_SECRET"
```

Comment:
added  `NEXT_PRIVATE_STRIPE_WEBHOOK_SECRET` to globalEnv due to an eslint error

Reply:
my bad